lab_1_keywords_tfidf/main.py

```diff
@@ -118,7 +181,31 @@ def calculate_expected_frequency(
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not doc_freqs:
+        return None
+    if not isinstance(doc_freqs, dict) or not isinstance(corpus_freqs, dict):
+        return None
+    for key, value in doc_freqs.items():
+        if not isinstance(key, str) or not isinstance(value, int):
+            return None
+    for key, value in corpus_freqs.items():
+        if not isinstance(key, str) or not isinstance(value, int):
+            return None
+    words_in_doc = sum(doc_freqs.values())
+    words_in_col = sum(corpus_freqs.values())
+    exp_freqs = {}
+    for key in doc_freqs:
+        l_val = words_in_doc - doc_freqs.get(key,0)
```

Comment:
why do you use `doc_freqs.get(key,0)`?

Reply:
consider using just doc_freqs[key]
can you think of why using `get` here is unnecessary?

Reply:
in case i iterate over doc_freqs keys, it is obvious that they exist, so there's no need to use get, right?